src/test/java/org/traccar/protocol/LaipacProtocolDecoderTest.java

```diff
@@ -120,6 +120,13 @@ public void testDecode() throws Exception {
         verifyPosition(decoder, text(
                 "$AVRMC,358174067149865,143747,R,5050.1124,N,00420.0542,E,1.34,161.96,190318,a,3416,119,1,0,0*7D"));
 
+        //Zero LAC, CID, MCC, MNC
```

Comment:
Space is missing after `//`.

Reply:
I used the same format used all around the file. No problem, I'll fix it all.

Reply:
I see. That's probably needs to be fixed as well. Someone else must have added it.